### DISCUSSION TITLE
How does "clone" work?

### DISCUSSION THREAD

**Discussion by ca0v** [viliusle/miniPaint#380] [Q&A]
I cannot make sense of the clone tool from a users perspective.  What is the tool supposed to do?

**Comment by viliusle** [comment]
Lets say you have nice photo of beach, background is sand, and there is something ugly on sand, you can use clone tool to erase it:

1. select clone tool
2. right click on area were sand is good
3. click on that ugly thing
4. done,

So basically it allows to you edit image, without creating new data, but using already existing area. Very powerful tool.
